### DISCUSSION TITLE
Puppy Linux (fossapup64-9.5) : The configuration failed to validate?

### DISCUSSION THREAD

**Discussion by davidjayjackson** [davatorium/rofi#1794] [Distro Specific]
![20230129_001918](https://user-images.githubusercontent.com/46544155/215306946-74eaf5e0-b8ba-4fda-8056-c6e8bda39d56.jpg)
 I am getting the same error with both i3 and xmonad?  When I try and run : "rofi -show run" I get the attached error? 
Thanks in advance,
David Jackson

I followed the instruction to create a config file:

 mkdir -p ~/.config/rofi
rofi -dump-config > ~/.config/rofi/config.rasi

**Comment by DaveDavenport** [comment]
that font name is indeed wrong. 
Can you attach the config and the output of `rofi -help`.

**Reply by davidjayjackson** [reply]
Thanks for following up.
Attached are the information you asked, output from help and config.

**Reply by DaveDavenport** [reply]
attached where?

**Reply by davidjayjackson** [reply]
[rofi.txt](https://github.com/davatorium/rofi/files/10530654/rofi.txt)
[rofi_help.txt](https://github.com/davatorium/rofi/files/10530655/rofi_help.txt)

**Reply by DaveDavenport** [reply]
you attached the help output twice, config is missing.

1.) You are on a very very very old (unsupported) version.
2.) The font format that is specified is unsupported
Because you did not post your config file, i  have no idea if you set it, or it is set at distro level.
See https://davatorium.github.io/rofi/current/rofi-theme.5/#font-parsing for the right format.
(It needs to be compatible with what pango expect).

**Reply by davidjayjackson** [reply]
Ok, 
Thanks again for your help.
I downloaded the lastest source.
What package is the configure script part of?
[config_rasi.txt](https://github.com/davatorium/rofi/files/10530850/config_rasi.txt)

**Reply by DaveDavenport** [reply]
It should all be explained here: https://davatorium.github.io/rofi/INSTALL/

If you are uncomfortable compiling, just fix the font string in your config. 1.5.4 is old, but should still work fine for most tasks.
See the previous link for the right font name structure.

**Reply by davidjayjackson** [reply]
I downloaded the source for 1.7.5.
Where is the configure script for 1.7.5

**Reply by davidjayjackson** [reply]
Good news , someone provided me with the link to your 1.7.5 source release.

**Reply by DaveDavenport** [reply]
The download links to the release sources are all under the releases link on the github page:
https://github.com/davatorium/rofi/releases/tag/1.7.5


'(the 'source code.tar.gz' links there I can regrettable not remove from release page, its a github thing
Also the INSTALL document explains how to generate the configure script
)

**Reply by davidjayjackson** [reply]
Thanks,
You can close this ticket.

**Reply by DaveDavenport** [reply]
what ticket?

**Comment by davidjayjackson** [comment]
1794

**Reply by DaveDavenport** [reply]
That is a discussion thread, and you can mark it as answered.
It has no 'close'.
